lib/tasks/assets.rake

```diff
@@ -0,0 +1,14 @@
+# Adapted from https://github.com/heroku/heroku-buildpack-ruby/issues/792
+namespace :assets do
+  desc "Remove 'node_modules' folder"
+  task rm_node_modules: :environment do
+    Rails.logger.info "Removing node_modules folder"
+    FileUtils.remove_dir("node_modules", true)
+  end
+end
+
+unless ENV["WEBPACKER_PRECOMPILE"] == "false"
+  Rake::Task["assets:clean"].enhance do
+    Rake::Task["assets:rm_node_modules"].invoke
+  end
```

Comment:
This is safe because `node_module` is not needed after webpack compilation. The would refer to the compiled js instead of the raw dependency.

I was able to verify that benhalpern-community still works after the `node_module` is removed.